modules/bazel-diff/7.2.14/source.json

```diff
@@ -0,0 +1,4 @@
+{
+    "url": "https://github.com/Tinder/bazel-diff/releases/download/7.2.14/release.tar",
```

Comment:
Is this file intentionally not compressed?

Reply:
Fixed